typescript/cli/src/deploy/warp.ts

```diff
@@ -243,9 +357,204 @@ async function writeDeploymentArtifacts(
   log(indentYamlOrJson(yamlStringify(warpCoreConfig, null, 2), 4));
 }
 
+async function resolveWarpIsmAndHook(
```

Comment:
The next three functions were also copied from the SDK. There must be a strong reason to move this code, and a plan to remove the duplication this PR causes.